### ISSUE TITLE
[Bug]: When `isPlaying=false` the image is blank instead of showing static image.

### ISSUE THREAD

**Issue by devjn** [alexzhirkevich/compottie#12]
### What happened?

When `isPlaying=false` the image is blank instead of showing static image. I have tested on Android 12. When using airbnb lottie the image is displayed.
Here is usage:
```
        Image(
            modifier = Modifier.padding(end = 16.dp).size(48.dp),
            painter = rememberLottiePainter(
                composition = composition,
                isPlaying = isAnimated,
                iterations = Compottie.IterateForever,
            ),
            contentDescription = "Lottie animation"
        )
```

### Unsupported features

- [ ] My animation doesn't use unsupported features

### What Compottie version do you experience the bug on?

2.0.0-beta02

### What platforms are you seeing the problem on?

Android

### Does Compottie log anythyng to the stdout / your logger set with `Compottie.logger`?

```shell
Not sure if it related anyhow, I get:
ImageReader_JNI  W : `Unable to acquire a buffer item, very likely client tried to acquire more than maxImages buffers`
```

**Comment by alexzhirkevich** [comment]
Thanks. Reproduced. Does the issue occurs only when progress is zero or in the middle of the animation too?

**Comment by devjn** [comment]
Even if stopped in the middle

**Comment by alexzhirkevich** [comment]
Can you share reproducible animation for the middle-case? I can only reproduce the first missing frame

**Comment by devjn** [comment]
Currently I am not able to share, I will try to find. But what I mean by stopping in the middle is that from my example I just update `isAnimated = false`. So maybe it is just reset to first frame?

**Comment by devjn** [comment]
You can use this example, I was able to reproduce with it too:
https://github.com/spemer/lottie-animations-json/blob/master/ic_fav/ic_fav.json

**Comment by alexzhirkevich** [comment]
Should be fixed in `2.0.0-rc01`

**Comment by devjn** [comment]
Yes it is fixed, thanks!
